alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java

```diff
@@ -171,7 +191,7 @@ public ImageMagickTransformationIT(final Pair<TestFileInfo, Pair<String,String>>
                 allTargets("quick.png"),
                 allTargets("quick.pnm"),
                 allTargets("quick.ppm"),
-                //allTargets("quick.psd"),
+                allTargets("quick.psd"),
                 //allTargets("quick.tiff"),
```

Comment:
I realise it might be out-of-scope for this JIRA but do we know why tiff is commented out ? 

Is it already raised in JIRA as tech debt for further investigation ?

Reply:
now quick.psd in not commented out anymore. However, searching "tiff" in JIRA returns 0 results so I am not sure why line 195 is there.

Reply:
I just tried to uncomment it and it creates a lot of new failing tests

Reply:
Hmm - ok, I suggest we add to the list for "legacy - part 2" for now ;-)